### PR TITLE
feat: add policy to assign reviewer

### PR DESCRIPTION
**Pull request by versoul**
## Description

Add policy to assign reviewer.

## Related issue

Closes [issue #164](https://github.com/reviewpad/reviewpad/issues/164)

## Type of change

New feature (non-breaking change which adds functionality)

## How was this tested?

Run in test config & enviropment
run tests

## Checklist

- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works 
- [x] I have ran `task check -f` and have no issues

## Code review and merge strategy (ship/show/ask) 

 
 
- [x] Ask: this pull request requires a code review before merge

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,6 +15,7 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/reviewpad/api/go v0.0.0-20220824160655-7aaf235bcfa2
 	github.com/reviewpad/go-conventionalcommits v0.10.0
+	github.com/reviewpad/host-event-handler v0.0.3
```
Comment (marcelosousa) [inline go.mod:18]: This package should not be used anymore.

**Review by marcelosousa — CHANGES_REQUESTED** [review]
Please rebase with the latest and remove dependencies of old host-event-handler package.

**Comment by versoul** [comment]
@marcelosousa done

**Comment by ferreiratiago** [comment]
Closing in favor of https://github.com/reviewpad/reviewpad/pull/382

**Pull request closed without merging**